README.md

```
@@ -187,6 +187,44 @@ kiichaind genesis collect-gentxs
+
+### Prerequisites
+- **Go**: Version 1.23.6+ ([Install Guide](https://golang.org/dl/))
+- **Git**: For cloning the repository
+- **Optional**: GNU Make for simplified builds
+- Check your setup:
+  ```bash
+  go version  # Should show 1.23.6+
+  git --version
+Setup StepsClone the Repository:bash
+
+git clone https://github.com/KiiChain/kiichain.git
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)